api/adscert.proto

```diff
@@ -58,11 +63,23 @@ message AuthenticatedConnectionVerificationRequest {
 }
 
 // AuthenticatedConnectionVerificationResponse contains the results of verifying
-// signatures.
+// signatures for a request.
 message AuthenticatedConnectionVerificationResponse {
     VerificationStatus verification_status = 1;
-    bool body_valid = 2;
-    bool url_valid = 3;
+    VerificationInfo verification_info = 2;
+}
+
+// AuthenticatedConnectionVerificationBatchRequest contains multiple requests for verifying
+// signatures generated by another party.
+message AuthenticatedConnectionVerificationBatchRequest {
+    repeated RequestInfo request_info = 1;
+}
+
+// AuthenticatedConnectionVerificationBatchResponse contains the results of verifying
+// signatures for multiple requests.
+message AuthenticatedConnectionVerificationBatchResponse {
+    VerificationStatus verification_status= 1;
```

Comment:
For batch verification, is the verification_status field still necessary at the top level, or can the individual verifications sufficiently communicate their individual status?

Reply:
This relates to issue #37 

The `SignatureStatus` and `VerificationStatus` status types refer to that specific operation rather than the actual result. For example, `VefificationStatus == OK` means the operation succeeded without errors but the `UrlValid` and `BodyValid` fields can still be false because the signature itself isn't valid.

Those status types should be renamed to say `ResponseStatus` or `OperationStatus` to be more clear about the intended meaning. And then there's the (very overloaded) `AuthenticatedConnectionProtocolStatus` which is used in parts for the signature (during signing) as well as domain indexing. We should split that apart and clean up those different states as well.

What do you think?